package.json

```diff
@@ -13,6 +13,9 @@
   "main": "index.js",
   "author": "Chimera Prime",
   "license": "MIT",
+  "scripts": {
+    "postversion": "git push --tags && yarn publish . --tag $npm_package_version && git push && echo \"Successfully released version $npm_package_version!\""
```

Comment:
shouldn't we release version where it's merged on master? right now if we update version it will be posted to npm registry, what if we have bugs in the release? i think it would be better to release to npm on master, and on release branches just `create` version and tag them

Reply:
The same if we push to the master and release version from master which includes bugs. Whe should talk about flow with the whole team.